haystack_experimental/components/extractors/llm_metadata_extractor.py

```diff
@@ -257,13 +241,15 @@ def from_dict(cls, data: Dict[str, Any]) -> "LLMMetadataExtractor":
 
 
     @component.output_types(documents=List[Document], errors=List[Tuple[str,Any]])
```

Comment:
```suggestion
    @component.output_types(documents=List[Document], errors=List[Optional[Tuple[str,Any]]])
```
That said, why do we need a document id at all in `errors` - looks `documents` and `errors` are parallel arrays, so each index should correspond.

Reply:
good point, I change it to remove the `id`